src/core/utils/modal.tsx

```diff
@@ -66,7 +66,12 @@ function Modal({
   };
 
   return (
-    <FocusTrap>
+    <FocusTrap
+      focusTrapOptions={{
+        // set fallbackFocus when running vitest to avoid focus trap errors
```

Comment:
This is a minor, but we usually format our comments with the first letter in uppercase `// Set fallbackFocus` and end the comment with a period `.`